arwen/elrondapi/elrondei.go

```diff
@@ -1380,15 +1381,12 @@ func v1_3_upgradeContract(
 		return
 	}
 
-	gasSchedule := metering.GasSchedule()
-	gasToUse = gasSchedule.ElrondAPICost.AsyncCallStep
-	metering.UseGas(gasToUse)
```

Comment:
API call cost should have stayed

Reply:
See `runtime.ExecuteAsyncCall()` below, it consumes the `AsyncCallStep` cost.